### PR TITLE
Add the fcgiproto project

### PR DESCRIPTION
**Pull request by agronholm**

### CODE REVIEW

File: index.rst [hunk 1]
```diff
@@ -48,12 +48,15 @@ Implementations
 =========================== =======
 Protocol                    Project
 =========================== =======
+`FastCGI`_                  fcgiproto_
```
Comment (Lukasa) [inline index.rst]: The URL markup here is broken.

**Comment by Lukasa** [comment]
I've got no problems with this, so long as we fix up the markup issue. @brettcannon?

**Comment by agronholm** [comment]
What markup issue are you referring to? The preview looked fine.

**Comment by agronholm** [comment]
Hopefully it should be fine now.

**Comment by Lukasa** [comment]
Ok, so long as @brettcannon is happy I'm happy.

**Comment by brettcannon** [comment]
Thanks, @agronholm !

**Pull request merged**